pulp_rpm/app/viewsets.py

```diff
@@ -303,3 +312,38 @@ def create(self, request):
             kwargs={}
         )
         return OperationPostponedResponse(async_result, request)
+
+
+class ModuleOneShotUpload(ContentViewSet):
+    """
+    ViewSet for One Shot Modulemd Upload.
+    """
+
+    endpoint_name = 'modulemd'
+    queryset = Modulemd.objects.all()
+    serializer_class = ModulemdSerializer
```

Comment:
Am I correct that this endpoint will create *multiple* modulemd from *one* modules.yaml file?

Reply:
yes it will create all modulemds which are listed in modulemd.yaml file.

Reply:
Since that's the case, I wonder if it should be called something that makes that more clear?